app/views/producer_mailer/order_cycle_report.text.haml

```diff
@@ -1,13 +1,13 @@
-#{t :producer_mail_greeting} #{@producer.name},
+#{t :producer_mail_greeting} #{raw(@producer.name)},
```

Comment:
It's worth noting because we're using HAML, which is a "HTML abstraction markup language". It's not designed for this plaintext format!
But I don't have a better suggestion.

The only thing I can think of is to use the [`!=`](https://haml.info/docs/yardoc/file.REFERENCE.html#unescaping_html) operator for full lines. But that doesn't work for this line, so it's better to just use `raw` consistently, as you have 👍